gutenberg.php

```diff
@@ -149,7 +149,7 @@ function gutenberg_init( $return, $post ) {
 		return $return;
 	}
 
-	if ( isset( $_GET['classic-editor'] ) ) {
+	if ( array_key_exists('classic-editor', $_GET ) ) {
```

Comment:
PHPCS fails here because a space is needed inside the opening parenthesis:

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage

Reply:
Can you explain why this change is necessary? This part of your original comment is not clear to me:

>When invoking the classic-editor no parameter value is passed so the isset() test returns false.

Based on this comment, I would expect that an unassigned value is still an empty string, and thus should return `true` for `isset`:

http://php.net/manual/en/reserved.variables.get.php#98419

Reply:
Further, if the logic is complex enough that it warrants a back-and-forth, we may want to extract this out to a shared utility method to share between here and the other use.

Reply:
From PHP manual for isset http://php.net/manual/en/function.isset.php
> isset — Determine if a variable is set and is not NULL

Reply:
I didn't think it was necessary to create a separate method to test for the classic-editor just to solve this problem. However, I've now found other uses of isset() against different locations ie. $_REQUEST as well as $_GET so I imagine there are other problems lurking. I'll investigate further.

Reply:
>From PHP manual for isset http://php.net/manual/en/function.isset.php
>
>>isset — Determine if a variable is set and is not NULL

To my original question, when would `$_GET['classic-editor']` ever be null?

Reply:
when you're not using the `classic-editor` plugin the URL is simply 

https://qw/src/wp-admin/post-new.php?classic-editor
or 
https://qw/src/wp-admin/post.php?post=629&action=edit&classic-editor

and that causes the value to be null.

I only installed the `classic-editor` plugin today.
This was the first time I saw a value being passed.

Reply:
>and that causes the value to be null.

But as was explained in the [previous link I shared](http://php.net/manual/en/reserved.variables.get.php#98419), this is not true.

When the URL is `?classic-editor`, the value is still an empty string:

![image](https://user-images.githubusercontent.com/1779930/35983019-29d3d87e-0cbf-11e8-89d8-50bad0d3ace5.png)